### PR TITLE
Add files to improve community health

### PR DESCRIPTION
**Pull request by brucellino**
Some files have been added to this repo, in the .github hidden directory
in order to improve the community health and help pass the corresponding
[inspec profile](https://github.com/brucellino/black-panther).

Added files include:
- code of conduct, with emphasis on how we deal with issues.
  Adapted from the contributor covenant
- contributing guidelines
- issue and pull request templates

this addresses #76

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2018 EGI Foundation
```
Comment (alvarolopez) [inline LICENSE.txt]: Is this correct?

Although a lot of code (almost all code?) does not have copyright headers there are a lot of contributions not being made by the EGI Foundation.

Comment (gwarf) [reply LICENSE.txt]: Hi @alvarolopez , it's something we took from another repo we have been working on, and indeed I should have reacted formally when reviewing this PR.
Let's discuss and find something appropriate.

Comment (gwarf) [reply LICENSE.txt]: @alvarolopez let's discuss it in #76.

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+- Feature or documentation requests, where they describe a need or gap
+- Authoring or review of releases
+- Direct authorship of code or documentation
+- Identifying and fixing bugs
+
+## Contribution Process
+
+Before proposing a contribution via pull request, please ensure that an issue is open describing the need for your contribution.
+You will need to refer to this issue number when you submit the pull request.
+- **It is recommended to make pull requests against release candidate branches, whenever features are involved**, instead of against the master branch. See [Release Cycle](#release-cycle) below.
```
[the remark this answers is not in the document]
Comment (gwarf) [reply .github/CONTRIBUTING.md]: Yes, we don't really use release candidate branches here, and we want to have PR against master.

[the remark this answers is not in the document]
Comment (gwarf) [reply .github/CONTRIBUTING.md]: And I think we want to have PR for everything, not sure about that "Obvious fix" thingy

**Pull request merged**